solidity/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keep-network/keep-core",
-  "version": "1.8.0-pre.0",
+  "version": "1.8.0-rc",
```

Comment:
This doesn't match `package.json`.

Reply:
Fixed in e6254d503bf7b6ab559501bffe6f4e7cd369788c.